eyeshade/workers/reports.js

```diff
@@ -957,38 +957,52 @@ const prepareReferralPayout = async (debug, runtime, authority, reportId, thresh
 
   const payments = []
   for (let i = 0; i < eligPublishers.length; i++) {
-    const payment = statements[eligPublishers[i].publisher].balance
+    const publisher = eligPublishers[i].publisher
+    const payment = statements[publisher].balance
+    console.log(payment)
     payment.type = 'referral'
-    payment.fees = payment.probi.times(feePercent).truncated()
-    payment.probi = payment.probi.minus(payment.fees)
+    payment.fees = '0'
+    // payment.probi = payment.probi
```

Comment:
can you remove this commented out assignment? also, I think we may want payment.fees to be a bignumber instead of a string. it probably doesn't matter because we call toString down the line but best to keep the types consistent.